.changesets/fix_bryn_internal_server_error.md

```diff
@@ -0,0 +1,8 @@
+### Internal server error handling change ([PR #5159](https://github.com/apollographql/router/pull/5159))
+
+When an http 500 is returned we should not return the details of the error to the client. 
+Instead, they are now logged at ERROR level.
+
+In addition, the error is now returned as a graphql error rather than a plaintext error, giving a better experience in sandbox.
+
```

Comment:
```suggestion
### 5xx Internal server error responses are structured errors ([PR #5159](https://github.com/apollographql/router/pull/5159))

An internal server error (5xx class) which occurs as a result of an unexpected/unrecoverable disruption to the GraphQL request lifecycle execution (e.g., a coprocessor failure, etc.) will now result in a structured GraphQL error (i.e., `{"errors": [...]}`) being returned to the client rather than a plain-text error as was the case previously.

When these circumstances occur, the underling error message — which may be any number of internal disruptions — will still be logged at an `ERROR` level to the router logs for the administrator of the router to monitor and use for debugging purposes.
```